init_test.go

```diff
@@ -78,7 +78,6 @@ func testSubMerchantAccount() string {
 
 func init() {
 	logEnabled := flag.Bool("log", false, "enables logging")
-	flag.Parse()
```

Comment:
This isolates the `logEnabled` so it will always be `false`.

Reply:
This raises an error documented here: https://github.com/golang/go/issues/31859 during initialization.